bin/add_c_nc.py

```diff
@@ -26,9 +26,6 @@ def add_c_nc(score, ref):
     bl = clin_nc.new_start.values
     bc = clin_nc.new_chr.values
 
-    bc[bc=='MT'] = -1 # Treat MT chromosome as NaN values to avoid string comparison
-    bc = bc.astype(int)
```

Comment:
I thought having this code is more accurate. Do you think it's better to revert this change?